presto-main/src/test/java/com/facebook/presto/operator/GroupByHashYieldAssertion.java

```diff
@@ -67,8 +68,8 @@ private GroupByHashYieldAssertion() {}
     }
 
     /**
-     * @param operatorFactory creates an Operator that should directly or indirectly contain GroupByHash
-     * @param getHashCapacity returns the hash table capacity for the input operator
+     * @param operatorFactory         creates an Operator that should directly or indirectly contain GroupByHash
```

Comment:
Are changes on this and the next line intended?

Reply:
It looks like it got changed by mistake when formatting, reverted it back